gemfile

```diff
@@ -1,3 +1,5 @@
+source 'https://rubygems.org'
+
```

Comment:
Want to be explicit about this, otherwise the following error can happen on other folks' systems:

```
$ bundle install
[DEPRECATED] This Gemfile does not include an explicit global source.
Not using an explicit global source may result in a different lockfile being
generated depending on the gems you have installed locally before
bundler is run. Instead, define a global source in your Gemfile like this:
  source "https://rubygems.org".
Could not find gem 'rspec' in locally installed gems.
```